Tests/js/models/assets/ez-fielderrordetails-tests.js

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
to be consistent when the path of the file being tested, this file should be in `Tests/js/models/structs/assets/ez-fielderrordetails-tests.js`